### PR TITLE
28-storage addition

### PR DESCRIPTION
**Pull request by jtpittman195**
Hey Ryan, New breakboot addition proposal, 28-storage.
John

### CODE REVIEW

**Comment by jtpittman195** [comment]
Sorry about all the commits.  Accident.

File: breaks/28-storage [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 # Description: Remove partition containing root logical volume
 
-REQUIRE_CANTBOOT_VERSION_MAGIC
+REQUIRE_CANTBOOT_VERSION_MAGIC 1
```
Comment (ryran) [inline breaks/28-storage]: That would actually need to be a "2" if you made changes to the main launcher.

**Comment by jtpittman195** [comment]
Thanks Ryan, I realized I made a mistake there this morning.
says it right there, overlooked
#We should increment this any time we add new functions
cantbootVersionMagic=1

**Comment by jtpittman195** [comment]
Ok Ryan, that's it for now.  A little confusing with the previous issues corrected regarding the magic number, sorry.  2 functions added on different commits so incremented to 3.  29-filesystem added and ext require funtion.

**Comment by ryran** [comment]
@jtpittman195 
Awesome!

I effectively just merged the 28-storage thing into master, via a jpittman195-storage28 feature branch. I wanted to (1) squash the ton of commits into one commit and (2) I wanted to make changes myself and test before pushing to master.

To make that easier in the future John (and to make it more obvious that the original code came from you), I'd greatly appreciate it if you work with feature branches, so the workflow would be:

```
# fork my repo
# clone your fork
git clone git@github.com:YOURNAME/cantboot.git
cd cantboot
git checkout -b NN-NAME

# make changes and commit
git push origin NN-NAME

# then you can test on a live machine with, e.g.,:
curl -Lko breakboot https://raw.githubusercontent.com/YOURNAME/cantboot/NN-NAME/breakboot
gitRoot=https://raw.githubusercontent.com/YOURNAME/cantboot/NN-NAME ./breakboot NN-NAME
```

Most of this was already in the README.md, but I just updated it with the explanation about using `gitRoot` environment variable to test, so hopefully others don't have as hard a time in the future. Thanks for bringing it to my attention! :)

I see you added a 29 one too. I'll take a look later tonight.

**Pull request closed without merging**